lib/core/timers/timer-api.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Timeout } from './timer';
+
+let binding = require('../../binding');
+
+export function setImmediate(func: (...args: any[]) => void, ...args: any[]): Timeout {
+    let timeout = new Timeout(func, 0, 0, args);
+    binding.setImmediate(timeout);
+    return timeout;
+}
+
+export function clearImmediate(immediate: Timeout): void {
```

Comment:
There should be a class `Immediate`.
https://nodejs.org/dist/latest-v8.x/docs/api/timers.html#timers_class_immediate

Reply:
Noticed issue. Add alias Immediate for Timeout.